src/main/kotlin/com/arsiu/eduhub/controller/CourseController.kt

```diff
@@ -32,6 +31,14 @@ class CourseController(
             HttpStatus.OK
         )
 
+    @GetMapping("/mostElements")
+    fun getAllCoursesSortedByInnerElements(): ResponseEntity<List<CourseDtoResponse>> =
+        ResponseEntity(
+            courseMapper.toDtoResponseList(courseService.sortCoursesByInners()),
+            HttpStatus.OK
+        )
```

Comment:
seems, it could be simplified:
```kotlin
    @GetMapping("/mostElements")
    fun getAllCoursesSortedByInnerElements(): List<CourseDtoResponse> =
            courseMapper.toDtoResponseList(courseService.sortCoursesByInners())
```

Reply:
done